gz_ros2_control/package.xml

```diff
@@ -11,10 +11,13 @@
 
   <depend>ament_index_cpp</depend>
   <!-- default version to use in official ROS2 packages is Ignition Fortress for ROS2 Rolling -->
-  <depend condition="$IGNITION_VERSION == ''">ignition-gazebo6</depend>
-  <depend condition="$IGNITION_VERSION == citadel">ignition-gazebo3</depend>
-  <depend condition="$IGNITION_VERSION == edifice">ignition-gazebo5</depend>
+  <depend condition="$IGNITION_VERSION == ''">gz-sim7</depend>
```

Comment:
We've been changing this to `GZ_VERSION`

Reply:
https://github.com/ros-controls/gz_ros2_control/pull/67/commits/1f011b322ff1d36a9eb0a539bb53b9b778aa3c82